k8s/mlflow/files/dbchecker.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+if [[ -z "${PGHOST}" && -z "${MYSQL_HOST}" ]]; then
```

Comment:
Thanks @Mnoble-19 
I would love to know more about this MYSQL setup. More like detailed notes about its future plans and all that. cc: @BenjaminSsempala

Reply:
> Thanks @Mnoble-19 I would love to know more about this MYSQL setup. More like detailed notes about its future plans and all that. cc: @BenjaminSsempala

Hello @Baalmart , no this is nothing to worry about. 
The bash script came with the helm chart and is for checking if one's  db is available, it's setup to check for either mysql or postgresql or both

in our scenario, we're using postgresql which is already setup on our VMs so no worries on mysql